kore/src/Kore/Reachability/Claim.hs

```diff
@@ -773,7 +773,7 @@ checkSimpleImplication inLeft inRight existentials =
         rhsBottom <-
             fmap isBottom . liftSimplifier $
                 SMT.Evaluator.filterMultiOr $srcLoc
-                    =<< Pattern.simplify right
```

Comment:
this didn't end up being the issue, but it's not correct and should be changed nonetheless.